<launch>
  <!-- ARGUMENT DEFINITIONS -->
  <arg name="vis" default="false" /> <!-- run rviz? -->
  
    <!-- startup simulator node -->
    <node pkg="sawyer_humcpp" type="applewalls" name="applewalls"
	  respawn="true" output="screen" />
    <node pkg="sawyer_humcpp" type="cursor_sm.py" name="cursor_sm"
	  respawn="true" output="screen" />
    
  <!-- rviz if needed -->
  <group if="$(arg vis)">
    <node pkg="rviz" type="rviz" respawn="false" name="rviz" 
	  args="-d $(find sawyer_humcpp)/launch/apple-view.rviz" />
  </group>

</launch>

  
